workflow_avamb/src/transfer_contigs_and_aggregate_all_nc_bins.py

```diff
@@ -237,24 +252,28 @@ def choose_best_ripped_bin_and_mv_if_nc(
             )
 
             # bin_A keeps the contigs
-            if cluster_A_complet > 90 and cluster_A_cont < 5:
+            if cluster_A_complet > min_comp and cluster_A_cont < max_cont:
```

Comment:
Another issue - not sure how important this is. If someone sets `min_comp = 0.9`, then I would expect 0.9 to also pass, so the check here should be `>=` not `>`. I.e. consider what happens if someone sets `min_comp = 1`.

Reply:
I completely agree, how can we proceed to implement the replacement of `>` by `>=` ? Should I make a new commit ?

Reply:
Yes. Just make new commits - I think the best is to squash them in the end (i.e. replace all commits in this entire PR with a single commit with all changes). GitHub can do this automatically.